tests/unit_tests/unit_monetization_api.test.ts

```diff
@@ -562,7 +562,7 @@
                             tierType: 'sortByCpm',
                             instances: [
                                 {
-                                    providerName: 'Vungle',
+                                    providerName: 'liftoffMonetize',
```

Comment:
The providerName should not be changed to 'liftoffMonetize'. Please revert this change.